Asterix/tests/test_testbed.py

```diff
@@ -39,7 +39,7 @@ def test_def_thd():
     optical_bench = Bench([entrance_pupil, dm3, corono], ["entrancepupil", "DM3", "corono"])
 
     testbed_psf = optical_bench.todetector_intensity()
-    assert np.max(testbed_psf) < 5e-6, "PSF after wrapped vortex without aberrration should be better than 5e-6"
+    assert np.max(testbed_psf) < 1e-8, "PSF after wrapped vortex without aberrration should be better than 5e-10"
```

Comment:
1e-8 or 5e-10? You use both on the same line here.

Reply:
corrected